pkg/nodejs/packagejson/parse.go

```diff
@@ -35,16 +35,26 @@ func (p *Parser) Parse(r io.Reader) (Package, error) {
 		return Package{}, xerrors.Errorf("JSON decode error: %w", err)
 	}
 
+	license := parseLicense(pkgJSON.License)
+
+	hasContent := license != "" || len(pkgJSON.Dependencies) > 0 || len(pkgJSON.OptionalDependencies) > 0
```

Comment:
it looks as separate function.
In the future we can add more options here(e.g. `workspaces`).